tests/python/rest_api/test_tasks.py

```diff
@@ -1012,6 +1012,71 @@ def test_datumaro_export_without_annotations_includes_image_info(
                     if "size" in related_image:
                         assert tuple(related_image["size"]) > (0, 0)
 
+    @pytest.mark.usefixtures("restore_db_per_function")
+    @pytest.mark.usefixtures("restore_redis_ondisk_per_function")
+    @pytest.mark.parametrize(
+        "format_name, num_frames, frame_step, expected_frames",
+        [
+            ("Datumaro 1.0", 100, 5, 20),
+            ("COCO 1.0", 100, 5, 20),
+            ("CVAT for video 1.1", 100, 5, 20),  # no remainder
+            ("CVAT for video 1.1", 97, 5, 20),  # prime
+            ("CVAT for video 1.1", 97, 2, 49),
+            ("CVAT for video 1.1", 100, 3, 34),  # three
+            # we assert that expected frames are ceil(frames / step)
+        ],
+    )
+    def test_export_with_non_default_frame_step(
+        self,
+        # fixtures
+        tmp_path: Path,
+        admin_user: str,
+        # params
+        format_name: str,
+        num_frames: int,
+        frame_step: int,
+        expected_frames: int,
```

Comment:
```suggestion
        expected_frame_count: int,
```
I think, it might be useful to check the actual frame names as well.